### PR TITLE
[Style/#91] 마이페이지 컨텐츠 내부 리뷰 남기기 UI 구현

### PR DESCRIPTION
**Pull request by SebellKo**
## 📌 작업 내용

리뷰 남기기 UI를 구현했습니다.

- [x] 리뷰 남기기 컴포넌트 UI 구현
- [x] 글래스 아이콘 클릭시 UI 변경 구현

<br/>

## 📌 구현 결과 (선택)

구현한 기능이 보이는 스크린샷을 업로드해주세요.
스크린샷 2024-08-22 오전 4 29 55

## 📌 기타 사항

<br/>

### CODE REVIEW

**Comment by JIMIN1020** [comment]
<h2>Coverage Report</h2>
<table> <thead> <tr> <th align="center">Status</th> <th align="left">Category</th> <th align="right">Percentage</th> <th align="right">Covered / Total</th> </tr> </thead> <tbody> <tr> <td align="center">🔵</td> <td align="left">Lines</td> <td align="right">9.98%</td> <td align="right">545 / 5460</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Statements</td> <td align="right">9.98%</td> <td align="right">545 / 5460</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Functions</td> <td align="right">17.25%</td> <td align="right">39 / 226</td> </tr> <tr> <td align="center">🔵</td> <td align="left">Branches</td> <td align="right">28.24%</td> <td align="right">74 / 262</td> </tr> </tbody> </table><details><summary>File Coverage</summary><table> <thead> <tr> <th align="left">File</th> <th align="right">Stmts</th> <th align="right">% Branch</th> <th align="right">% Funcs</th> <th align="right">% Lines</th> <th align="left">Uncovered Lines</th> </tr> </thead> <tbody> <tr> <td align="left" colspan="6"><b>Changed Files</b></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/Review.tsx">src/features/SideMenu/components/Review/Review.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/Review.tsx#L1-L2" class="text-red">1-2</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/Review.tsx#L4-L5" class="text-red">4-5</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/Review.tsx#L7-L8" class="text-red">7-8</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/Review.tsx#L10-L18" class="text-red">10-18</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/Review.tsx#L20-L22" class="text-red">20-22</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/Review.tsx#L24" class="text-red">24</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/Review.tsx#L26" class="text-red">26</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx">src/features/SideMenu/components/Review/ReviewForm.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L1-L8" class="text-red">1-8</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L10-L13" class="text-red">10-13</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L15-L17" class="text-red">15-17</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L19-L31" class="text-red">19-31</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L34-L39" class="text-red">34-39</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L41-L50" class="text-red">41-50</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L52-L60" class="text-red">52-60</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L62-L63" class="text-red">62-63</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L65" class="text-red">65</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L67" class="text-red">67</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L73" class="text-red">73</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L80" class="text-red">80</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L83-L85" class="text-red">83-85</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L88" class="text-red">88</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L92" class="text-red">92</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L99" class="text-red">99</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L105-L107" class="text-red">105-107</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L110" class="text-red">110</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L120-L122" class="text-red">120-122</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/features/SideMenu/components/Review/ReviewForm.tsx#L125" class="text-red">125</a></td> </tr> <tr> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/pages/MainPage.tsx">src/pages/MainPage.tsx</a></td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="right">0%</td> <td align="left"><a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/pages/MainPage.tsx#L1-L6" class="text-red">1-6</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/pages/MainPage.tsx#L8-L9" class="text-red">8-9</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/pages/MainPage.tsx#L11-L14" class="text-red">11-14</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/pages/MainPage.tsx#L16-L31" class="text-red">16-31</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/pages/MainPage.tsx#L33" class="text-red">33</a>, <a href="https://github.com/Licruit/Licruit-frontend/blob/8cf657af7cc60461af07fd7e53119a37b72ce99d/src/pages/MainPage.tsx#L35" class="text-red">35</a></td> </tr> </tbody> </table></details>
<em>Generated in workflow <a href=https://github.com/Licruit/Licruit-frontend/actions/runs/10496243042>#133</a></em>

File: src/features/SideMenu/components/Review/ReviewForm.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import { useForm } from 'react-hook-form';
+import { useState } from 'react';
+import { INPUT } from '../../constants/input';
+import Label from '../common/Label';
+import ProfileInput from '../common/ProfileInput';
+
+function ReviewForm() {
+  const [rate, setRate] = useState<number>(1);
+  const theme = useTheme();
+  const methods = useForm();
```
Comment (JIMIN1020) [inline src/features/SideMenu/components/Review/ReviewForm.tsx:13]: 타입 지정 안해도 되나요?

**Review by JIMIN1020 — APPROVED** [review]
고생 많으셨습니다~~!

**Review by juhee067 — APPROVED** [review]
세종님 수고많으셨습니다. 파일 이름도 싸악 정리하셨네요
고생하셨습니다.

File: src/features/SideMenu/components/Review/ReviewForm.tsx (hunk 1 above)
Comment (SebellKo) [reply src/features/SideMenu/components/Review/ReviewForm.tsx:13]: 아아 요거 api 연동하면서 하려고 했어요 ~

**Pull request merged**